Expand definition of cosmic space, add definitions of sigma-space, aleph-space and aleph_0-space

creator1creeper1 opened the pull request:
This depends on #508.

Expand the definition of cosmic space and add definitions of $\sigma$-space, $\aleph$-space and $\aleph_0$-space. The latter 3 terms are all defined analogously to and in the same reference as the first term, so if the first term is relevant enough to be included, I think the latter 3 terms are relevant enough as well.

prabau commented:
Not commenting on the merit of the mathematics here, but as far as the procedure for submitting PRs is concerned, it's preferable not to introduce dependencies between different PRs.  Specifically, the first four commits here are really independent of the rest and should not have been added here as they are already in #508.  In this case it's no big deal, but in general having the PRs independent of each other makes them easier to review and merge into main in any order.

prabau commented:
This shows 29 files being changed, but in fact only a few are relevant here.  Oh well...

prabau commented:
Thanks very much for your useful contribution.  Here are a few comments.

1) P74 (cosmic).  The stated definition of network seems overly complicated.  All you need is that every open set is a union of a subfamily of $\mathcal P$.  Also no need to say that $\mathcal P$ is a cover of $X$, as that follows automatically.  Maybe say something like: "A family $\mathcal P$ of subsets of $X$ is called a network if ..."

2) P177 (sigma-space) Need similar simplification for notion of network.  On the other hand, it could be helpful a newcomer to spell out what a sigma-locally finite collection means (= countable union of locally finite families).  Or is that too much? (I think "locally finite" is well-known and does not need to be explained further).

3) Are you sure double backslashes are needed in the names of the last three properties?

4) P178, P179: simplify a little (phrase k-network starting with a family of subsets, no need to specify cover.

5) IMPORTANT: Should the four definitions assume the space is $T_3$?  E. Michael, P. O'Meara who introduced the notions assume that.  The referenced article (Ikeda & Tanaka) don't say so explicitly in their definitions, but at the end of their Introduction they say "We assume that spaces are regular $T_1$".  Encyclopedia of General Topology also assumes that.

@StevenClontz Any comments on that?

creator1creeper1 commented:
Thank you very much for the quick feedback!

The reason I made the PRs depend on each other was that they both touched the same file, so I thought it would be easier to merge them after each other if I made the second one include the commits of the first one. I wanted to split them up into separate PRs since I thought that changing some file formatting and actually adding new content was sufficiently different to justify two separate PRs. However, I see now that this complicated the PR process, and in the future I will simply submit one PR with all the changes.

Since I rebased this branch on the main branch, now it should show only the 4 files that are actually affected.

1. I reformulated the definition of network to reflect your feedback.

2. I reformulated the definition of network and spelled out the definition of a sigma-locally finite collection.

3. Yes, doubly backslashes are necessary. This is also done, for example, in property 17 ($\sigma$-compact)

4. I simplified the definition accordingly.

5. I've added the $T_3$ requirement.

prabau commented on properties/P000177.md:
```diff
@@ -0,0 +1,8 @@
+  - mr: 1305126
+    name: Spaces having star-countable k-Networks
+---
+A family $\mathcal{P}$ of subsets of $X$ is called a network if for any open set $U \subseteq X$ there is some subfamily $\mathcal{P}' \subseteq \mathcal{P}$ with $U = \bigcup \mathcal{P}'$. A family of subsets of $X$ is called $\sigma$-locally finite if it is a countable union of locally finite familes. $X$ is called a $\sigma$-space if it is $T_3$ and has a $\sigma$-locally finite network.
```
typo: familes --> families

prabau commented:
Maybe the double backslashes are not needed.  See for example P132, P114, etc.

StevenClontz commented:
(I'm at JMM this week, hope to put in some hours reviewing here early next week.)

creator1creeper1 commented:
* Fixed the typo with "familes" -> "families"
* Regarding the double backslashes: I've figured out that if one wraps the name of a property in quotation marks, one needs double backslashes for the latex commands. Per your suggestion, I've removed the quotation marks and the double backslashes.

prabau commented:
Good find about the backslashes.
